CMakeLists.txt

```diff
@@ -0,0 +1,109 @@
+cmake_minimum_required(VERSION 3.5)

```

Comment:
Since this project is being converted to CMake, I suggest to use the latest `3.12.1`

Reply:
I'll change CMake version, good point.

Regarding project name, I thought so too, but then it occurred to me that most CMake-packaged libs omit "lib" in their name - you type `find_library(curl)` instead of `find_library(libcurl)`. Also, IIRC if you tell cmake to `find_library(blah)` it will look for `libblah.a` automatically, so `find_library(libbacktrace)` would result inf looking for `liblibbacktrace.a`. I also wonder, for which package is FindBacktrace designed? What do you think about my points?